doc/classes/Object.xml

```diff
@@ -59,6 +59,21 @@
 				Sets a property. Returns [code]true[/code] if the [code]property[/code] exists.
 			</description>
 		</method>
+		<method name="_to_string">
+			<return type="String">
+			</return>
+			<description>
+				Returns a [String] representing the object.
```

Comment:
I'd mention here that this is the method to override if you want to customize the String representation of the object, which will be used both when calling `[method to_string]` manually but also in `[method @GDScript.print]` and similar.